### PR TITLE
Feature/steps box shadow

### PR DESCRIPTION
**Pull request by Akswii**

### CODE REVIEW

File: src/less/components/steps.less [hunk 1]
```diff
@@ -30,7 +29,6 @@
 
             &:hover {
                 background-color: @nearwhite;
-                box-shadow: 1px 0 0 0 @gray-bright, -1px 0 0 0 @gray-bright, 0 1px 0 0 @nearwhite;
```
Comment (hmosvoll) [inline src/less/components/steps.less]: Is this intentional? This is a 1px line above and below a step item, a border-like presentation that does not effect the box model.

File: src/less/components/steps.less [hunk 2]
```diff
@@ -126,7 +124,6 @@
 
     .steps-selected {
         background-color: @nearwhite;
-        box-shadow: 1px 0 0 0 @gray-bright, -1px 0 0 0 @gray-bright, 0 1px 0 0 @nearwhite;
```
Comment (hmosvoll) [inline src/less/components/steps.less]: Same question.

File: src/less/components/steps.less [hunk 3]
```diff
@@ -205,10 +202,6 @@
             text-align: left;
             padding: @list-item-padding-desktop-vertical;
 
-            &.steps-selected {
-                box-shadow: 0 1px 0 0 @gray-bright, 0 -1px 0 0 @gray-bright;
-            }
```
Comment (hmosvoll) [inline src/less/components/steps.less]: Same question.

File: src/less/components/steps.less (hunk 1 above)
Comment (Akswii) [reply src/less/components/steps.less]: I think i went through the component too fast and just removed anything related to box-shadow. I added the commented sections again.

File: src/less/components/steps.less [hunk 4]
```diff
@@ -205,14 +204,14 @@
...
-
             > a {
                 padding: @list-item-padding-desktop-vertical;
                 width: auto;
                 .negative-margin(@list-item-padding-desktop-vertical);
+
+                &.steps-selected {
```
Comment (hmosvoll) [inline src/less/components/steps.less]: Why have you moved the `.steps-selected` class from the `li`element to the anchor element here?

Comment (Akswii) [reply src/less/components/steps.less]: ..just an error. Fixed it and double checked it, should be okay now!

**Pull request merged**